### ISSUE TITLE
Offline app won't start

### ISSUE THREAD

**Issue by craig-at-rsg** [sarriaroman/FabricPlugin#25]
My app doesn't initialize correctly when it's offline and this plugin is included, at least on iOS. Is being online a requirement for this plugin's initialization?

**Comment by Justin-Credible** [comment]
Can you clarify what you mean by "doesn't initialize properly"? Is the app crashing?

The plugin code itself doesn't depend on internet access, but I can't say for certain what the Fabric SDK requirements are.

**Comment by craig-at-rsg** [comment]
I did some more testing, and I'm not so sure I identified the problem correctly. Basically, my testing looked conclusive, but I misinterpreted the results. 

Originally, I tried running with and without this Fabric plugin installed, and I seemed to be stuck on a white screen (my webview didn't finish loading) only with it installed.

My current testing, with the device wifi and cell data off, allowed my app to start up quickly with or without this plugin installed. With the wifi on using an unreliable network, I was stuck on the white screen with and without this plugin installed. 

I'm sorry for the noise.
